clowdapp.yaml

```diff
@@ -66,9 +66,6 @@ objects:
           value: "${PATH_PREFIX}"
         - name: APP_NAME
           value: "${APP_NAME}"
-        # Should be a secret ?
-        - name: SECRET_KEY_BASE
```

Comment:
Wouldn't Rails shout when this is not defined?

Reply:
Hmm, it might be :see_no_evil: I'll check next week

Reply:
Yeah, we'll just mark this as a placeholder in the clowdapp :+1: